config-service-impl/src/main/java/org/hypertrace/config/service/ConfigResource.java

```diff
@@ -0,0 +1,12 @@
+package org.hypertrace.config.service;
+
+import lombok.Data;
+
+@Data
```

Comment:
Consider making this immutable with `@Value` - I didn't notice anywhere where we were mutating it, which is good.

Reply:
This was immutable previously also(declared as `final`). But still replaced it with `@Value` to make it a bit compact.